Tests/Integration/IntegrationTestFixtures.swift

```diff
@@ -84,6 +84,41 @@ extension IntegrationTestFixtures {
             transportProtocol: transportProtocol).get()
     }
 
+    static func createMobileCoinClientConfigWithPartialValidConsensusUrls(transportProtocol: TransportProtocol) throws -> MobileCoinClient.Config {
+        try MobileCoinClient.Config.make(
+            consensusUrls: ["mc://invalid.mobilecoin.com", network.consensusUrl],
+            consensusAttestation: network.consensusAttestation(),
+            fogUrls: [network.fogUrl],
+            fogViewAttestation: network.fogViewAttestation(),
+            fogKeyImageAttestation: network.fogLedgerAttestation(),
+            fogMerkleProofAttestation: network.fogLedgerAttestation(),
+            fogReportAttestation: network.fogReportAttestation(),
+            transportProtocol: transportProtocol).get()
+    }
+    
+    static func createMobileCoinClientConfigWithPartialValidFogUrls(transportProtocol: TransportProtocol) throws -> MobileCoinClient.Config {
+        try MobileCoinClient.Config.make(
+            consensusUrls: [network.consensusUrl],
+            consensusAttestation: network.consensusAttestation(),
+            fogUrls: [network.fogUrl, "fog://invalid.mobilecoin.com"],
+//            fogUrls: ["fog://invalid.mobilecoin.com"],
```

Comment:
move this string into one of the fixtures files as a constant

Reply:
yup - the tests were WIP, will do... tx